src/components/ColorPicker/__stories__/colorPicker.stories.js

```diff
@@ -60,6 +62,35 @@ export const NoColorIcons = () => {
   );
 };
 
+export const MultiSelect = () => {
```

Comment:
If user can't play with the multi select because it needs a wrapper with state management delete this story